src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-infra-hosts.mdx

```diff
@@ -0,0 +1,241 @@
+        `processors::memory_limited`
+      </td>
+      <td>
+        The [memory limiter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/memorylimiterprocessor) processor is used to prevent out of memory situations on the collector. Given that the amount and type of data the collector processes is environment specific and resource utilization of the collector is also dependent on the configured processors, it is important to put checks in place regarding memory usage.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `processors::cumulative_delta`
+      </td>
+      <td>
+        The [cumulative delta](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) processor converts cumulative sum metrics to cumulative delta which is recommended in order to query system metric rates more easily in New Relic One.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `service::pipelines::metrics`
+      </td>
+      <td>
+        Make sure `hostreceiver` and `resourcedetection` are included.
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+Here is a sample configuration YAML file. Be sure to do the following:
+
+  * Replace <var>OTLP_ENDPOINT_HERE</var> with the appropriate [endpoint](#review-settings.
+  * Replace <var>YOUR_KEY_HERE</var> with your account's [license key](https://one.newrelic.com/launcher/api-keys-ui.launcher).
+
+```
+extensions:
+  health_check:
+
+receivers:
+  hostmetrics:
+    collection_interval: 20s
+    scrapers:
+      cpu:
+        metrics:
+          system.cpu.utilization:
+            enabled: true
+      load:
+      memory:
+        metrics:
+          system.memory.utilization:
+            enabled: true
+      disk:
+      filesystem:
+        metrics:
+          system.filesystem.utilization:
+            enabled: true
+      network:
+      paging:
+        metrics:
+          system.paging.utilization:
+            enabled: true
+      processes:
+      process:
+
+processors:
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 1000
+    spike_limit_mib: 200
+  batch:
+  cumulativetodelta:
+    metrics:
+      - system.network.io
+      - system.disk.operations
+      - system.network.dropped
+      - system.network.packets
+      - process.cpu.time
+  resource:
+    attributes:
+      - key: host.id
+        from_attribute: host.name
+        action: upsert
+  resourcedetection:
+    detectors: [env, system]
+
+exporters:
+  otlp:
+    endpoint: <var>OTLP_ENDPOINT_HERE</var>
+    headers:
+      api-key: <var>YOUR_KEY_HERE</var>
+
+service:
+  pipelines:
+    metrics:
+      receivers: [hostmetrics]
+      processors: [batch, resourcedetection, resource, cumulativetodelta]
+      exporters: [otlp]
+
+  extensions: [health_check]
+```
+
+## Step 3: View your data [#using-ui]
+
+You can view your collector data in a variety of places in the New Relic UI.
+
+### Browsing host data in infrastructure UI [#using-ui]
```

Comment:
anchor name on lines 200 and 204 are the same

Reply:
@barbnewrelic, good catch! I  just fixed this!